app/src/test/shared/services/cache/SingletonVariableCache.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from "chai";
+import { singletonVariableCache } from "../../../../main/shared/services/cache/singletonVariableCache";
+
+describe('Singleton variable cache tests', () => {
+
+    it('sets and gets data', () => {
+        const key = "test.test1.test2";
+        singletonVariableCache.clearAll();
+        expect(singletonVariableCache.get(key)).to.equal(undefined);
```

Comment:
I'm finding this test hard to follow - could you split it up into separate tests with one or two `expect` statements each?